app/pages/project/vpcs/VpcsPage.tsx

```diff
@@ -120,6 +122,12 @@ export function VpcsPage() {
     <>
       <PageHeader>
         <PageTitle icon={<Networking24Icon />}>VPCs</PageTitle>
+        <DocsPopover
+          heading="VPCs"
+          icon={<Networking16Icon />}
+          summary="VPCs are private networks that serve to isolate sets of instances from each other. Instances within a VPC can talk to each other over private IP addresses (if firewall rules allow it) but communication between VPCs must happen through external IPs."
```

Comment:
Was able to cut a line by cutting a few words. Also "using private IP addresses" feels slightly more correct than "over private IP addresses"

> VPCs are private networks that isolate sets of instances from each other. Instances within a VPC can talk to each other using private IP addresses (if firewall rules allow it) but traffic between VPCs must go through external IPs.

<img width="420" alt="image" src="https://github.com/oxidecomputer/console/assets/3612203/548acb8f-0929-4767-b86e-ef07608dde28">